### ISSUE TITLE
Build: Router Table Box Joint Jig/Sled

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-cad-cam-cnc#67]
The project I am currently working on uses Plywood that is too dense to cut on the laser. 
So I'm building it with more "traditional" woodworking methods.
I'm _manually_ cutting box joint "fingers" using a router for accuracy/consistency instead of hand-sawing them.

Need to create a JIG/Sled to control the passes on the router table. 
Hopefully it will be re-useable for future projects using the same material.

**Comment by nelsonic** [comment]
This was my  first attempt:

![image](https://user-images.githubusercontent.com/194400/189543790-3b45e65f-028c-4127-bdb6-7f1fa646bb13.png)

While I was designing it I realized that I could make it _much_ better so I'm starting again from scratch now.

**Comment by nelsonic** [comment]
_Really_ want to finish the project associated with this Jig. But cannot work on it remotely so back on the Backlog. ⏳

**Comment by nelsonic** [comment]
Never have "spare" time ... but this project *neeeeds* to be a priority. 💭 
Here's my second draft:
![image](https://user-images.githubusercontent.com/194400/199307862-cac02a14-eed7-4275-8251-e49163ae2655.png)

Cutting it on the laser now.

**Comment by nelsonic** [comment]
Cut on laser:
![A5D55F07-CD92-49E2-A14A-FA113F699B8C](https://user-images.githubusercontent.com/194400/199351495-e24d7c7b-ccf3-4d80-b7ae-869b766be8a7.jpeg)

QR Code works even at an angle:
![F8DB47ED-84C7-4B83-9143-8A0B10C3ABAD](https://user-images.githubusercontent.com/194400/199351628-e657edc0-eef2-449a-91ad-2506d77f5e6f.jpeg)

assembly:

![C89DD958-425A-4A0F-AD0A-D58BD78785D3](https://user-images.githubusercontent.com/194400/199351664-2f901c93-3568-441a-bfca-a4a12ce92a8d.png)

**Comment by nelsonic** [comment]
Fully assembled box joint jig/sled on Trend CRT router table: 

![0361E985-2A6B-468A-AD6D-DC28FAFDCB7B](https://user-images.githubusercontent.com/194400/199351710-97533dab-0f6a-4e4e-833c-f2e26f51a3f9.jpeg)

ready for testing tomorrow. 🤞🏼
Too much noise to run the router table tonight. Going to get an “early” (22:30) night instead and do some code/computer work early in the morning. ☀️

**Comment by nelsonic** [comment]
First attempt at a box joint using my new jig:
Used 15mm birch ply scrap for the test. 
![34A0C83E-EC7B-4843-96BC-CD83CC223A8E](https://user-images.githubusercontent.com/194400/199525284-f0ba20cb-e81d-4a23-a09d-eb46597b3772.jpeg)


![3A933F05-8AEB-47CA-8824-4E3B307D86F7](https://user-images.githubusercontent.com/194400/199525127-a45afa7e-9d07-4938-94f9-05f1eb3f9dc4.jpeg)

this is a ***DRY*** fit. The space you see is where the glue will go. It will be a near-perfect fit. Not bad. 👌

**Comment by nelsonic** [comment]
Sadly, when I did the second test with a sample of the more dense plywood, it misaligned:

![image](https://user-images.githubusercontent.com/194400/199601460-f07c7387-b74e-490a-be34-7e67dfe3c635.jpeg)

Difficult to see in this image. But basically the jig is a bust because the error compounds.

But I know how to fix it. It's just going to take a bit of effort. 💭⏳

**Comment by nelsonic** [comment]
Need to do a bit of code work this evening to unblock others so will need to come back to this on the weekend. 🤞🏼

**Comment by nelsonic** [comment]
This: https://youtu.be/LE1Wixu7E-g

**Comment by nelsonic** [comment]
Second iteration:

![image](https://user-images.githubusercontent.com/194400/199795554-1201f099-6597-4905-a6d6-045206b487a8.jpeg)

_flawless_ box joint. 👌

**Comment by nelsonic** [comment]
This is the box joint in its "sharp" edge form:

![image](https://user-images.githubusercontent.com/194400/200021518-1b7b8a7e-ac22-4135-b151-b44aa2de0512.jpeg)

And here it is with a rounded edge:

![image](https://user-images.githubusercontent.com/194400/200021591-a95b055e-1e09-4142-9c85-1735643d89a7.jpeg)

I don't know which I prefer...
I tend to do round edges on my projects because they look "finished" to me and cause fewer injuries when people bump into them in real life. 
But the sharp edge of the box joint looks good IMO ... 😍

What do *you* think? 💭

**Comment by nelsonic** [comment]
https://www.goodreads.com/quotes/831377-to-plant-a-garden-is-to-believe-in-tomorrow

**Comment by nelsonic** [comment]
Full size box joint:

 ![image](https://user-images.githubusercontent.com/194400/200431007-aafa7149-c712-411e-abd1-50223f83d6fa.jpeg)

Quite pleased with this. 👌
Don't worry, it's nowhere near "finished". Still a _lot_ of work to do. Currently dry-fitting it. 👷

**Comment by nelsonic** [comment]
the end result was "OK" but the Box Joint Jig left a lot to be desired.
If (when?) I'm able to make time to re-do it I will use ***`metal`*** not **`wood`** for the Jig. 😉  ref: #106

**Comment by nelsonic** [comment]
Unpinning the this issue as it's not a priority right now. But will return to it when I have more time ...

**Comment by nelsonic** [comment]
I tried to build a second version of this `Jig` to use it for an Xmas Gift ... ⏳ 
Sadly, I ran out of time because of Thumbageddon https://github.com/nelsonic/nelsonic.github.io/issues/917 👎 
Having to park this again to focus on higher priority tasks. 🅿️
